killbill/api/account_api.py

```diff
@@ -0,0 +1,5229 @@
+# coding: utf-8
+
+"""
+    Kill Bill
+
+    Kill Bill is an open-source billing and payments platform  # noqa: E501
+
+    OpenAPI spec version: 1.0.0
+    
+    Generated by: https://github.com/swagger-api/swagger-codegen.git
+"""
+
+
+from __future__ import absolute_import
+
+import re  # noqa: F401
+
+# python 2 and python 3 compatibility library
+import six
+
+from killbill.api_client import ApiClient
+
+
+class AccountApi(object):
+    """NOTE: This class is auto generated by the swagger code generator program.
+
+    Do not edit the class manually.
+    Ref: https://github.com/swagger-api/swagger-codegen
+    """
+
+    def __init__(self, api_client=None):
+        if api_client is None:
+            api_client = ApiClient()
+        self.api_client = api_client
+
+    def add_account_blocking_state(self, account_id, body, x_killbill_created_by, x_killbill_api_key, x_killbill_api_secret, **kwargs):  # noqa: E501
```

Comment:
`x_killbill_created_by, x_killbill_api_key, x_killbill_api_secret` should become keyword arguments, i.e. something like:

```
...created_by=None, api_key=None, api_secret=None, **kwargs
```

(let's also drop the `x_killbill_` prefix)

Reply:
@pierre @sbrossie there is a way to edit parameters name using the `api.mustache` file ?

Reply:
@wwjfrodriguez By default i suppose the generator will use whatever name is defined in the swagger schema. Now the swagger model to generate the code is the following:
1. There are hooks in place for you to write custom code and update/augment whatever info comes from swagger schema
2. The Mustache template engine is being passed your template and the object map -- that your code in step 1. had a chance to tweak as needed.

So, short story, you could either replace the original parameter name with something you like and add other variable that your template uses to generate the code.  For instance in the java generator i am tweaking the parameter map that is passed to the template engine, and you can wee that i could [here](https://github.com/killbill/killbill-swagger-coden/blob/master/src/main/java/org/killbill/billing/codegen/languages/KillbillJavaGenerator.java#L414) modify parameter name if i wanted to.

So, something easy you could do is to look for any parameter  `x_killbill_xx` and change the name to be `xx` -- or create another field, that you use in your templates. Makes sense?

Reply:
Ok, So what do you prefer? Change manually the parameters name using the IDE (find/replace) or try to develop a custom model that can rename the parameters?

Reply:
Definitely the second one: The generated code should be kept 100% untouched -- otherwise this is unmaintainable. However, it is fine to have additional non generated code -- in different files -- if needed. 

It is very likely that in the same way i created a generator [KillbillJavaGenerator](https://github.com/killbill/killbill-swagger-coden/blob/master/src/main/java/org/killbill/billing/codegen/languages/KillbillJavaGenerator.java) for the java generation, we will need a `KillbillPythonGenerator`. The code generation works then revolve between adjusting templates and custom generator (`KillbillPythonGenerator`) to produce what we need.

Reply:
Ok, I'll be developing a `KillbillPythonGenerator`